cluster/bin/snappy-job.sh

```diff
@@ -127,7 +127,7 @@ while (( "$#" )); do
         echo "The netrc file $passwordfile not found."
         exit 1
       fi
-      securePart=" --netrc-file ${passwordfile}"
+      securePart=" --config ${passwordfile}"
```

Comment:
The other comment should explain it https://github.com/SnappyDataInc/snappydata/pull/763#discussion_r132102194